docker/prod/prod.template.yaml

```diff
@@ -20,10 +26,34 @@ spec:
       containers:
         - name: nginx
           image: artifactory.wikia-inc.com/sus/mediawiki-prod-nginx:${IMAGE_TAG}
+          livenessProbe:
+            httpGet:
+              path: /health/check
+              port: 8080
+            initialDelaySeconds: 60
+            timeoutSeconds: 5
+            periodSeconds: 20
+          readinessProbe:
+            httpGet:
+              path: /health/check
+              port: 8080
+            initialDelaySeconds: 5
```

Comment:
`readinessProbe. initialDelaySeconds` will be executed after `livenessProbe.initialDelaySeconds` so it will be 65 seconds until pod is ready for traffic. Is it intended?

Reply:
image pull/container creation takes around 1 minute so I think this should be oK

Reply:
But I think the timer starts when image is already pulled.